ed/idlpatches/NoInterfaceObject.patch

```diff
@@ -0,0 +1,151 @@
+From b4ddf97e8642974a2b4a7f6910309dd950c44710 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Philip=20J=C3=A4genstedt?= <philip@foolip.org>
+Date: Tue, 13 Apr 2021 09:44:13 +0200
+Subject: [PATCH] Replace NoInterfaceObject with LegacyNoInterfaceObject
+
```

Comment:
It would be tedious to list individual pull requests, but perhaps a link to check which ones remain?

```suggestion

https://github.com/KhronosGroup/WebGL/pulls?q=is%3Apr+is%3Aopen+nointerfaceobject

```

Reply:
Good idea! I see those also add `Exposed=Window`, so it would be better if my patch here did that too to avoid future conflicts.

Reply:
OK, done!